onedal/datatypes/validation.py

```diff
@@ -279,7 +299,7 @@ def _check_n_features(self, X, reset):
             f"is expecting {self.n_features_in_} features as input.")
 
 
-def _num_features(X):
+def _num_features(X, fallback_1d=False):
```

Comment:
This is changes in common primitives. Should be on separate PRs

Reply:
Discussed. Decided that it is correct